src/utils.jl

```diff
@@ -301,7 +301,7 @@ end
     """ ->
     function gplot(sm::SimResult)
         N = length(sm.colnames)
-        figure()
+        Gaston.figure()
```

Comment:
Yes, for some reason the call to figure needs to be fully qualified.
Actually, when I first started using Sims, Gaston was broken so I used
Winston, now it seems to be the other way around :-)

On Wed, Feb 18, 2015 at 9:59 AM, Tom Short <notifications@github.com> wrote:

> In src/utils.jl
> <https://github.com/tshort/Sims.jl/pull/57#discussion_r24923577>:
>
> > @@ -301,7 +301,7 @@ end
> >      """ ->
> >      function gplot(sm::SimResult)
> >          N = length(sm.colnames)
> > -        figure()
> > +        Gaston.figure()
>
> I see you used Gaston. That must mean it still works (after your fix)! I
> wasn't sure.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tshort/Sims.jl/pull/57/files#r24923577>.
>